Sources/xcresultparser/XCResultFormatter.swift

```diff
@@ -374,7 +386,15 @@ public struct XCResultFormatter {
 
 extension ActionTestMetadata {
     var isFailed: Bool {
-        return testStatus != "Success"
+        return isSuccessful == false && isSkipped == false
```

Comment:
This was the substantive change.  `isSuccessful` and `isSkipped` look for specific string values and any others are considered to be a failure.

Ideally this would be an enum in XCResultKit and we could switch on it.